tests/foreman/cli/test_hostgroup.py

```diff
@@ -268,7 +267,7 @@ def test_positive_update_hostgroup(
         def _cleanup():
             with session_puppet_enabled_sat:
                 HostGroup.delete({'id': hostgroup['id']})
-                capsule_cleanup(new_content_source['id'])
+                Proxy.delete({'id': new_content_source['id']})
```

Comment:
Could (and maybe should) also use the API here
```suggestion
                session_puppet_enabled_sat.cli.Proxy.delete({'id': new_content_source['id']})
```